addons/udes_warehouse_classification/tests/test_product_template.py

```diff
@@ -0,0 +1,62 @@
+from .common import Base
+
+
+class TestWarehouseClassificationPicking(Base):
+    """
+        # For a given product find the classification messages for specified report,
+        # 1. report a on it's own - banana
+        # 2. not matching report type - banana
+        # 3. report a and b - honeydew
+        # 4. report a and report a on inbound (i.e. 2 messages for a report) - cherry
+        # 5. no classifications - apple
```

Comment:
5 tests in docstring but 6 below! did one get added?

Reply:
fixed